numpy/core/src/umath/loops.c.src

```diff
@@ -1619,6 +1619,29 @@ TIMEDELTA_mm_m_remainder(char **args, npy_intp *dimensions, npy_intp *steps, voi
     }
 }
 
+NPY_NO_EXPORT void
+TIMEDELTA_mm_q_floor_divide(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    BINARY_LOOP {
+        const npy_timedelta in1 = *(npy_timedelta *)ip1;
+        const npy_timedelta in2 = *(npy_timedelta *)ip2;
+        if (in1 == NPY_DATETIME_NAT ||
+            in2 == NPY_DATETIME_NAT ||
+            in2 == 0) {
```

Comment:
nit: I would add an extra tab of indentation on these lines, to clearly separate the condition from the body below.

Reply:
the conditional is on a single line now after addressing Eric's request for `invalid`